src/Chainweb/TreeDB.hs

```diff
@@ -389,6 +390,10 @@ root :: TreeDb db => db -> IO (DbEntry db)
 root db = fromJuste <$> entries db Nothing (Just 1) Nothing Nothing S.head_
 {-# INLINE root #-}
 
+minRank :: TreeDb db => db -> IO Natural
```

Comment:
Please don't resolve a review question without an answer. `minRank` is unused outside of this module, so you need to justify the export, as if it were to not be used later in this module, it would be dead code.

Reply:
I think, the review asked for a Haddock comment, that I added before marking it resolved.

Reply:
It's exported because `TreeDB` exports and API, and `minRank` fits conceptually in that API. There is `maxEntry, `maxRank`, and `root` (`minEntry`). It makes sense to also have `minRank`.